clients/privacy-center/components/Logo.tsx

```diff
@@ -0,0 +1,18 @@
+import { Image } from "@fidesui/react";
+import React from "react";
+
+type LogoProps = {
+  logo_path: string;
+};
+
+const Logo: React.FC<LogoProps> = ({ logo_path }) => (
+  <Image
```

Comment:
it might be nice to have the `Logo` component itself take an optional link, and let it be responsible for determining if it renders as a link logo or a regular logo. that would be useful if we wanted this logo with linking ability elsewhere, and it'd keep `Layout.tsx` a little cleaner. just a thought, certainly not blocking!

it could look something like...

```tsx
// Layout.tsx
<Logo href={config.logo_url} src={config.logo_path} /> // in general I like to name props after similar props (`a` takes an `href`, and `Image` takes a `src`)

// Logo.tsx
type LogoProps = {
  src: string;
  href?: string;
}

// basically what you currently have for Logo
const LogoImage = ({src}:Pick<LogoProps, 'src'>) => (
  <Image src={src} ... />
)

// simple wrapper around LogoImage
const Logo = ({src, href}:LogoProps) => {
  
{ href? (
      <a href={href}>
        <LogoImage src={src} />
      </a>
      ) : (
        <LogoImage src={src} />
      )}
}

export default Logo;
```

Reply:
I really like this @allisonking - thank you!!

Reply:
resolved in https://github.com/ethyca/fides/pull/4090/commits/e486399aec3270808f09d162e0c955d753d6c764